package.json

```diff
@@ -54,6 +54,11 @@
                     "default": false,
                     "description": "(EXPERIMENTAL) Sync cursor position to vscode when vim quits"
                 },
+                "open-in-vim.integratedShellPath": {
+                    "type": "string",
+                    "default": null,
+                    "definition": "Shell path of integrated terminal."
+                },
```

Comment:
I wonder if this should be removed. Why would a user want to change `integratedShellPath`?

Reply:
On windows, we can change the git installation directory.😅
![ScreenClip](https://user-images.githubusercontent.com/4396864/68075716-6cbad600-fde6-11e9-8e08-95c3bdaf0468.png)

Reply:
That makes sense. Good point :smile: